daisy_workflows/image_import/debian/translate.py

```diff
@@ -67,10 +67,14 @@ def DistroSpecific(g):
       logging.warn('Could not uninstall Azure agent. Continuing anyway.')
 
     utils.update_apt(g)
-    utils.install_apt_package(g, 'google-cloud-packages-archive-keyring')
-    utils.install_apt_package(g, 'google-cloud-sdk')
-    utils.install_apt_package(g, 'google-compute-engine')
-    utils.install_apt_package(g, 'google-guest-agent')
+    pkgs = ['google-cloud-packages-archive-keyring', 'google-cloud-sdk',
+            'google-compute-engine']
+    # Debian 8 doesn't support the new guest agent, so we need to install
+    # the legacy Python version.
+    if deb_release == 'jessie':
+      pkgs += ['python-google-compute-engine',
+               'python3-google-compute-engine']
```

Comment:
For Debian 8, these two have to be installed on top of 'google-compute-engine'?

Reply:
@hopkiw confirmed that 'google-compute-engine' doesn't have a dependency in Debian 8 pointing to them. (Also, the Debian 8 tests fail without them)